### ISSUE TITLE
Loading multi prompt with config set in command line doesn't work

### ISSUE THREAD

**Issue by mocyuto** [promptfoo/promptfoo#359]
In [document](https://promptfoo.dev/docs/guides/gemini-vs-gpt#step-2-set-up-the-prompts), prompt can be set as map like below.

```yaml
prompts:
  prompts/gpt_prompt.json: gpt_prompt
  prompts/gemini_prompt.json: gemini_prompt
```

However, this doesn not work when specify at command line like below.

```console
promptfoo eval -c xxx.yaml
```

And error log is below
```console
/home/runner/.npm/_npx/81bbc6515d992ace/node_modules/promptfoo/dist/src/util.js:153
            prompts.push(...config.prompts);
                                   ^

TypeError: Found non-callable @@iterator
    at /home/runner/.npm/_npx/81bbc6515d992ace/node_modules/promptfoo/dist/src/util.js:153:21
    at Array.forEach (<anonymous>)
    at readConfigs (/home/runner/.npm/_npx/81bbc6515d992ace/node_modules/promptfoo/dist/src/util.js:148:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Command.<anonymous> (/home/runner/.npm/_npx/81bbc6515d992ace/node_modules/promptfoo/dist/src/main.js:237:26)
```

Maybe it is allowed only Array or string type, not Map type [here](https://github.com/promptfoo/promptfoo/blob/main/src/util.ts#L141-L147)

**Comment by typpo** [comment]
Thanks for flagging this, you're right the problem is that the code doesn't handle map type.

**Comment by typpo** [comment]
Fix is in #375

**Comment by typpo** [comment]
Fix published in 0.35.0
